BOJ/2468.안전 영역/6047198844.cpp

```diff
@@ -0,0 +1,67 @@
+#include <iostream>
+#include <algorithm>
+#include <cstring>
+#include <queue>
+
+using namespace std;
+
+int board[100][100];
+bool board_01[100][100];
+
+//�¿����
+int dx[4] = { -1,1,0,0 };
+int dy[4] = { 0,0,-1,1 };
+bool range(int n, int y, int x) {
+	if (0 > y || y >= n)
+		return false;
+	if (0 > x || x >= n)
+		return false;
+	return true;
+}
+
+int bfs(int under_value,int n) {
+	for (int y = 0; y < n; y++)
+		for (int x = 0; x < n; x++)
+				board_01[y][x] = board[y][x] <= under_value ? false : true;
```

Comment:
```suggestion
				board_01[y][x] = (board[y][x] > under_value);
```
논리연산의 결과물을 `bool` 타입에 집어넣는 것이니, 3항 연산자를 쓰지 않는 방법도 있습니다.

Reply:
와ㅏ.. 감사합니다.. 그러네요